apps/omg_db/test/fixtures.exs

```diff
@@ -24,7 +24,8 @@ defmodule OMG.DB.Fixtures do
     db_path = Briefly.create!(directory: true)
     Application.put_env(:omg_db, :path, db_path, persistent: true)
 
-    :ok = OMG.DB.init()
+    _ = OMG.DB.init()
```

Comment:
Don't care about `:ok` anymore?

Reply:
Good catch! I'd say we should care. Leftover change perhaps?